mapper/mapper.go

```diff
@@ -302,3 +326,7 @@ func CreateEditionsList(d dataset.Model, editions []dataset.Edition, datasetID s
 func convertMMMYYToTime(input string) (t time.Time, err error) {
 	return time.Parse("Jan-06", input)
 }
+
+func convertYYToTime(input string) (t time.Time, err error) {
```

Comment:
Very minor but I think this possibly should be 
```
convertYYYYToTime
```

because we're converting a full year string (i.e. `2016`) where as YY would suggest we're converting a 2-digit yeat (i.e. `16`).